nexus_capabilities/src/capabilities/plan_motion.cpp

```diff
@@ -120,6 +120,17 @@ make_request()
   req->max_velocity_scaling_factor = scale_speed;
   req->max_acceleration_scaling_factor = scale_speed;
 
+  auto maybe_read_only =
+    this->getInput<bool>("force_cache_mode_execute_read_only");
```

Comment:
The port should be defined in `providedPorts` (in the header) as well.

Reply:
Oops: d8f4cda74c1b854f48e579ad8f06355cc3f88525